### ISSUE TITLE
ng-view

### ISSUE THREAD

**Issue by kdaziz** [ggriffiths/MyOrange#1]
Is there a reason it looks like we are not actually using ng-view but hiding/displaying divs instead in the nav ctrl? I think I've got a fix for it but I wanted to check if it was actually broken first

**Comment by ggriffiths** [comment]
Yeah, it looks like we're not using it, but you'll notice that removing the ng-view div will cause everything to not work. Not really an expert on angular yet, but it seems to initialize the code in js/app.js that hides/shows each part.

If you have a fix go for it, but I think the issue has to do with us using normal Google APIs instead of an angular directive for each API. i.e. https://github.com/angular-ui/ui-map . I found out about this issue after we started, and I figured it was probably a little late to switch over.

**Comment by kdaziz** [comment]
My fix will make integration nothing more than adding the proper <script> tag to //client/main.html and adding the module to the myApp module dependency list.
Everything for a persons module can be contained in their directory and as long as //client/modules/schedule/index.html (for example) displays correctly, //client/main.html#/schedule should as well (scaled to fit ng-view of course).

It's working with the exception of one small bug due to some sort of css collision in degree plan but ill push it once i sort that out.

**Comment by kdaziz** [comment]
Lessons learned so far this semester: angular can be a temperamental little bitch
